FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/FhirToDataLakeProcessingJob.cs

```diff
@@ -77,12 +81,14 @@ public class FhirToDataLakeProcessingJob : IJob
             _fhirSchemaManager = EnsureArg.IsNotNull(fhirSchemaManager, nameof(fhirSchemaManager));
             _groupMemberExtractor = EnsureArg.IsNotNull(groupMemberExtractor, nameof(groupMemberExtractor));
             _filterManager = EnsureArg.IsNotNull(filterManager, nameof(filterManager));
+            _diagnosticLogger = EnsureArg.IsNotNull(diagnosticLogger, nameof(diagnosticLogger));
             _logger = EnsureArg.IsNotNull(logger, nameof(logger));
         }
 
         // the processing job status is never set to failed or cancelled.
         public async Task<string> ExecuteAsync(IProgress<string> progress, CancellationToken cancellationToken)
         {
+            _diagnosticLogger.LogInformation($"Start executing processing job {_inputData.ProcessingJobSequenceId}.");
```

Comment:
use `_jobId` instead of `_inputData.ProcessingJobSequenceId`, and same for others

Reply:
Removed information of job category. (Orchestrator or processing)